### DISCUSSION TITLE
Intensity Minutes Missing

### DISCUSSION THREAD

**Discussion by CalebJRogers** [philosowaffle/peloton-to-garmin#387] [Q&A]
Hello all,

First time user here as I just got my first Garmin watch. I was able to get everything installed easily via Docker - the detailed documentation made it so easy!

One thing I have noticed though is that my synced rides do not include "Intensity Minutes." I've attached a screenshot for reference. 

I've looked through all the settings/configuration (i'm running the webui version) and I don't "think" I've missed anything. I did not record the workout via my watch, or broadcast the heart rate - I used an external ANT+ heart rate monitor and just synced afterword. 

What is the best practice for making sure all the data is synced? I was planning on trying the watch as a heart rate broadcast next but wanted to check in here first to see what is suggested.

![IMG_0525](https://user-images.githubusercontent.com/53285077/206916781-2ae45869-eebb-43af-9832-9e1863ec4dfd.jpg)

**Comment by philosowaffle** [comment]
Intensity Minutes is something that is normally calculated by the watch so by default the Peloton data will not include Intensity Minute information.

Garmin seems to have documentation about what goes into [calculating intensity minutes](https://support.garmin.com/en-US/?faq=pNU9nnDzzGAHmEavp9rpY8), so theoretically P2G may be able to replicate what the Garmin watch is doing by measuring time in HR zones. But this would be a new feature.

Went ahead and migrated this to a feature request #388, so I can keep track of this idea and find some time to look into it.

**Reply by CalebJRogers** [reply]
Just to follow up: I broadcasted my HR from my watch to the peloton tonight. Everything syncs correctly AND I get intensity minutes.

![IMG-0531](https://user-images.githubusercontent.com/53285077/207992478-2802d3b4-d1d3-4f02-8571-28b5eed312eb.PNG)
